docs/source/services/timelock_service/migration.rst

```diff
@@ -0,0 +1,110 @@
+.. _timelock-migration:
+
+Migration to External Timelock Services
+=======================================
+
+.. danger::
+
+   Improperly configuring one's cluster to use external timestamp and lock services can result in **SEVERE DATA
+   CORRUPTION**! Please contact the AtlasDB team before attempting a migration.
```

Comment:
This one's intentional (the last few words of the warning are aimed specifically at the feature being documented)